src/ui-test/tests/context.menu.test.ts

```diff
@@ -2,51 +2,87 @@ import { expect } from 'chai';
 import * as path from 'path';
 import {
     ActivityBar,
+    after,
     before,
+    EditorView,
     SideBarView,
     ViewControl,
     ViewItem,
+    ViewSection,
     VSBrowser,
+    WebDriver,
 } from 'vscode-extension-tester';
+import {
+    CAMEL_ROUTE_YAML_WITH_SPACE,
+    CAMEL_RUN_ACTION_LABEL,
+    CAMEL_RUN_DEBUG_ACTION_LABEL,
+    TEST_ARRAY_RUN,
+    TEST_ARRAY_RUN_DEBUG,
+    disconnectDebugger,
+    executeCommandInContextMenu,
+    killTerminal,
+    waitUntilTerminalHasText
+} from '../utils';
 
 (process.platform === 'darwin' ? describe.skip : describe)('Camel file context menu test', function () {
+    this.timeout(240000);
+
+    let driver: WebDriver;
 
-    const CAMEL_RUN_DEBUG_MENU_ITEM = 'Run Camel Application with JBang and Debug';
-    const CAMEL_RUN_MENU_ITEM = 'Run Camel Application with JBang';
-    const CAMEL_ROUTE_YAML = 'demo route.camel.yaml';
+    before('Before setup', async function () {
+        driver = VSBrowser.instance.driver;
+    });
 
-    let item: ViewItem;
+    after('After cleanup', async function () {
+        await new EditorView().closeAllEditors();
+    });
 
-    describe('Camel Group items', function () {
-        this.timeout(180000);
+    beforeEach('Before each test', async function () {
+        await VSBrowser.instance.openResources(path.resolve('src', 'ui-test', 'resources'));
+        await VSBrowser.instance.waitForWorkbench();
 
-        before('Before setup', async function () {
-            await VSBrowser.instance.openResources(path.resolve('src', 'ui-test', 'resources'));
-            await VSBrowser.instance.waitForWorkbench();
+        await (await new ActivityBar().getViewControl('Explorer') as ViewControl).openView();
 
-            await (await new ActivityBar().getViewControl('Explorer') as ViewControl).openView();
+        const section = await new SideBarView().getContent().getSection('resources') as ViewSection;
+        await section.openItem(CAMEL_ROUTE_YAML_WITH_SPACE);
 
-            item = await (await new SideBarView().getContent().getSection('resources')).findItem(CAMEL_ROUTE_YAML) as ViewItem;
-        });
+        const editorView = new EditorView();
+        await driver.wait(async function () {
+            return (await editorView.getOpenEditorTitles()).find(title => title === CAMEL_ROUTE_YAML_WITH_SPACE);
```

Comment:
optional:

you do not need to define here editorView variable but you can directly use inline notation `await new EditorView().getOpenEditorTitles()`